Panic while fetching env

```
panic: runtime error: index out of range [-1]
--
  |  
  | goroutine 1 [running]:
  | github.com/buildkite/elastic-ci-stack-s3-secrets-hooks/s3secrets-helper/v2/secrets.handleEnvs(0xc00001e28f, 0x2d, 0xc00001e1e3, 0x1d, 0xc000018528, 0x11, 0xac2940, 0xc000461b48, 0xc00007c5a0, 0xac53e8, ...)
  | /buildkite/s3_secrets/s3secrets-helper/secrets/secrets.go:183 +0x5cd
  | github.com/buildkite/elastic-ci-stack-s3-secrets-hooks/s3secrets-helper/v2/secrets.Run(0xc00001e28f, 0x2d, 0xc00001e1e3, 0x1d, 0xc000018528, 0x11, 0xac2940, 0xc000461b48, 0xc00007c5a0, 0xac53e8, ...)
  | /buildkite/s3_secrets/s3secrets-helper/secrets/secrets.go:86 +0x505
  | main.mainWithError(0xc00007c5a0, 0xc00007c5a0, 0xc00019ff78)
  | /buildkite/s3_secrets/s3secrets-helper/main.go:54 +0x365
  | main.main()
  | /buildkite/s3_secrets/s3secrets-helper/main.go:23 +0x89
```

We had an empty `env` file. This made `r.data`'s len `0` (on the line this panics).

--- a/s3secrets-helper/secrets/secrets.go
+++ b/s3secrets-helper/secrets/secrets.go
@@ -180,12 +180,15 @@ func handleEnvs(conf Config, results <-chan getResult) error {
 			continue
 		}
 		data := r.data
-		if data[len(data)-1] != '\n' {
-			data = append(data, '\n')
-		}
-		log.Printf("Loading %s/%s (%d bytes) of env", r.bucket, r.key, len(r.data))
-		if _, err := bytes.NewReader(data).WriteTo(conf.EnvSink); err != nil {
-			return fmt.Errorf("copying env: %w", err)
+
+		if len(data) > 0 {
+			if data[len(data)-1] != '\n' {
+				data = append(data, '\n')
+			}
+			log.Printf("Loading %s/%s (%d bytes) of env", r.bucket, r.key, len(r.data))
+			if _, err := bytes.NewReader(data).WriteTo(conf.EnvSink); err != nil {
+				return fmt.Errorf("copying env: %w", err)
+			}
 		}
 	}
 	return nil

